backend/package.json

```diff
@@ -44,6 +44,6 @@
     "prettier": "2.8.7",
     "ts-jest": "29.0.5",
     "tsx": "^3.12.6",
-    "typescript": "^5.0.2"
+    "typescript": "^4.3.0"
```

Comment:
was there a reason for this change?

Reply:
yeah i think latest versioning is the practice we follow coz of the github bot settings.

Reply:
it's been done btw